cumulusci/core/tasks.py

```diff
@@ -151,7 +151,7 @@ def _process_exception(self, e):
 
     def _run_task(self):
         """ Subclasses should override to provide their implementation """
-        pass
+        raise NotImplementedError("Subclasses should provide their own implementation")
```

Comment:
I'm surprised this didn't cause any test failures, I sorta assumed we were using that implicit pass *somewhere* in the tests! awesome that we weren't.

anyway, this exception message doesn't provide anymore information that raising NotImplementedError bare does...

Reply:
I just copied it from what we're using elsewhere in this file.